### PR TITLE
Update publish-arc2 workflow to use right path.

### PR DESCRIPTION
**Pull request by TingluoHuang**
- Use `charts` instead of `charts_preview` as we moved the files to the `charts` folder
- Publish 2 different charts:
   - actions-runner-controller-2
   - auto-scaling-runner-set

### CODE REVIEW

File: .github/workflows/publish-arc2.yaml [hunk 1]
```diff
@@ -124,15 +124,19 @@ jobs:
...
+
+          ACTIONS_RUNNER_CONTROLLER_2_CHART_VERSION_TAG=$(cat charts/actions-runner-controller-2/Chart.yaml | grep version: | cut -d " " -f 2)
+          echo "ACTIONS_RUNNER_CONTROLLER_2_CHART_VERSION_TAG=${ACTIONS_RUNNER_CONTROLLER_2_CHART_VERSION_TAG}" >> $GITHUB_ENV
+          helm package charts/actions-runner-controller-2/ --version="${ACTIONS_RUNNER_CONTROLLER_2_CHART_VERSION_TAG}"
+          helm push actions-runner-controller-2-"${ACTIONS_RUNNER_CONTROLLER_2_CHART_VERSION_TAG}".tgz oci://ghcr.io/${{ steps.resolve_parameters.outputs.repository_owner }}/actions-runner-controller-charts
+
+          AUTO_SCALING_RUNNER_SET_CHART_VERSION_TAG=$(cat charts/auto-scaling-runner-set/Chart.yaml | grep version: | cut -d " " -f 2)
+          echo "AUTO_SCALING_RUNNER_SET_CHART_VERSION_TAG=${AUTO_SCALING_RUNNER_SET_CHART_VERSION_TAG}" >> $GITHUB_ENV
+          helm package charts/auto-scaling-runner-set/ --version="${AUTO_SCALING_RUNNER_SET_CHART_VERSION_TAG}"
+          helm push auto-scaling-runner-set-"${AUTO_SCALING_RUNNER_SET_CHART_VERSION_TAG}".tgz oci://ghcr.io/${{ steps.resolve_parameters.outputs.repository_owner }}/actions-runner-controller-charts
```
Comment (Link-) [inline .github/workflows/publish-arc2.yaml:186]: - Is there a scenario where we might want to publish them separately? 
- Is there an argument against having these split into 2 separate jobs?

Comment (TingluoHuang) [reply .github/workflows/publish-arc2.yaml:186]: They can be published separately if there is a chart-only bug (template render issues or anything not touching the CRD) in one of those charts.

I started with separate jobs, but to help PR review (reduce diff) I merged them into one for now.

I am willing to break them into 2 jobs if you want. 😄

Comment (Link-) [reply .github/workflows/publish-arc2.yaml:186]: Let's separate them if you have the time, we can iterate on this further at a later point

Comment (TingluoHuang) [reply .github/workflows/publish-arc2.yaml:186]: Updated to 2 jobs.

File: .github/workflows/publish-arc2.yaml [hunk 2]
```diff
@@ -124,21 +129,67 @@ jobs:
...
+          echo "short_sha=$(git rev-parse --short $resolvedRef)" >> $GITHUB_OUTPUT
+          echo "INFO: Normalizing repository name (lowercase)"
+          echo "repository_owner=$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_OUTPUT 
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3.3
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Publish new helm chart for actions-runner-controller-2 and auto-scaling-runner-set
```
Comment (cory-miller) [inline .github/workflows/publish-arc2.yaml]:
```suggestion
      - name: Publish new helm chart for auto-scaling-runner-set
```

**Review by cory-miller — DISMISSED** [review]

**Review by thboop — APPROVED** [review]
lgtm

**Review by cory-miller — APPROVED** [review]

**Pull request merged**